README.md

```diff
@@ -20,6 +20,33 @@ What's this? This is a template repository that sets up a few minor systems for
 
 ## FAQs
 
+### Some lint is unnecessary :( How do I disable it?
+There are actually 2 main ways to disable lint. Disabling the "rule" entirely, or just a single instance/in a file!
+#### Disabling the rule entirely.
+** **Make sure this is what you really want!! It is often likely that you want to just disable for a single file.** **
+
+Depending on whether it's from `stylelint` or `eslint`, you can go to `stylelintrc.json` and add to `"rules"
+```
+<rule-name>: null
+```
+or `eslintrc.json` and add
+```
+'<rule-name>': 'off',
+```
+#### Disabling a single instance/in a single file
```

Comment:
Not sure if I’m misunderstanding something or typo for the slash in 'instance/in'.